### PR TITLE
Milestone 1: Create data model

### PR DESCRIPTION
**Pull request by Christianib003**
- [x] `Initialiazed` rails app linked with postgresql.
- [x] Created `User` model and table.
- [x] Created `Comments`  model and table.
- [x] Created `Post` model and table.
- [x] Created `Likes` model and table.
- [x] `Add foreign keys` to connect the tables.

### CODE REVIEW

File: db/migrate/20221123133750_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :authorId
+      t.integer :postId
```
Comment (TedLivist) [inline db/migrate/20221123133750_create_comments.rb]: - [x] Kindly avoid using camel cases for your attribute names. Based on convention this should be the following. Kindly fix that.

  ```
  t.integer :author_id
  t.integer :post_id
  ```

- [x] Ensure these changes are reflected in the `schema.rb` file by running necessary migrations.

File: db/migrate/20221123134106_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorId
```
Comment (TedLivist) [inline db/migrate/20221123134106_create_posts.rb]: - [x] Kindly change to `author_id`. Ensure you run the migrations and reflect this change in the schema file.

File: db/migrate/20221123134106_create_posts.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      t.string :title
+      t.string :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```
Comment (TedLivist) [inline db/migrate/20221123134106_create_posts.rb]: - [x] Kindly change to `:comments_counter` and `:likes_counter`.

File: db/migrate/20221123134326_create_likes.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorId
+      t.integer :postId
```
Comment (TedLivist) [inline db/migrate/20221123134326_create_likes.rb]: - [x] These should be `:author_id` and `:post_id` respectively. Kindly fix that.

- [x] These changes should be reflect in the schema file.

File: db/migrate/20221123133423_create_users.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postsCounter
```
Comment (TedLivist) [inline db/migrate/20221123133423_create_users.rb]: - [x] The attribute should be `:posts_counter`. Kindly fix that.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# README
```
Comment (TedLivist) [inline README.md]: - [x] Kindly format your README using [the Microverse template](https://github.com/microverseinc/readme-template)

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Christianib003,

Good job so far!
✔️ PostgreSQL is used for DB
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Christianib003,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
approval1
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**